nft_ingester/src/metrics.rs

```diff
@@ -40,15 +40,19 @@ pub fn setup_metrics(config: &IngesterConfig) {
     }
 }
 
+// Returns a boolean indicating whether the redis message should be ACK'd.
+// If the message is not ACK'd, it will be retried as long as it is under the retry limit.
 pub fn capture_result(
     id: String,
     stream: &str,
     label: (&str, &str),
     tries: usize,
     res: Result<(), IngesterError>,
     proc: Instant,
-) -> Option<String> {
-    let mut ret_id = None;
+    txn_sig: Option<&str>,
+    account: Option<String>,
+) -> bool {
+    let mut should_ack = false;
```

Comment:
seems `let should_ack = match res { ... }` might be cleaner? 

That would let rust compiler also handle any potential branches in the match res thing that doesnt set should_ack by accident?

Also shoul_ack doesnt need to be mutable then.

Reply:
Yep, that's good feedback. I agree.